src/components/molecules/RadioGroupField/RadioGroupField.tsx

```diff
@@ -20,11 +21,27 @@ export interface RadioGroupFieldProps {
   label: string;
   items: RadioGroupFieldItem[];
   onChange: (value: string) => void;
+  onBlur?: () => void;
   disabled?: boolean;
   value?: string;
+  isValid?: boolean;
+  errorMessage?: string;
 }
 
-const RadioGroupField = ({ items, label, onChange, value, disabled }: RadioGroupFieldProps) => {
+const FieldError = styled(ErrorMessage)`
+  margin-top: 5px;
```

Comment:
maybe we could use `<ErrorMessage className="mt-5">` and avoid creating another component? _Same for RadioWrapper_

Reply:
on it 👍